test/school_test.rb

```diff
@@ -45,11 +40,34 @@ def test_it_can_add_student_names
   end
 
   def test_it_can_calculate_end_time
-    skip
     school1 = School.new('9:00', 7)
     school2 = School.new('9:00', 3)
 
     assert_equal '16:00', school1.end_time
     assert_equal '12:00', school2.end_time
   end
+
+  # Iteration 3 Tests:
+  def test_is_full_time?
+    school = School.new('9:00', 7)
+
+    assert_equal true, school.is_full_time?
+  end
+
+  def test_standard_student_names
+    school = School.new('9:00', 7)
+
+    school.add_student_name('Aurora')
+    school.add_student_name('tim')
+    school.add_student_name('megan')
+
+    assert_equal ['Aurora', 'Tim', 'Megan'], school.standard_student_names
+  end
+
+  # Iteration 4 Tests:
+  def test_convert_end_time_to_clock_time
+    school = School.new('9:00', 7)
+
+    assert_equal "4:00", school.convert_end_time_to_clock_time
```

Comment:
Similar to my note above, when we have conditional logic, we want to confirm all possible outcomes.  So, I would create a school where the end time was before noon to confirm that functionality as well.